catalog/resource_connection.go

```diff
@@ -39,6 +37,11 @@ var sensitiveOptions = []string{"user", "password", "personalAccessToken", "acce
 func ResourceConnection() *schema.Resource {
 	s := common.StructToSchema(ConnectionInfo{},
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
+			m["owner"] = &schema.Schema{
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: `owner field is deprecated due the Catalog API changes. Owner cannot be specified when creating a connection.`,
+			}
```

Comment:
@nkvuong pointed that we shouldn't remove Owner - we just need to set it via a separate API call

Reply:
Added the resource back. As discussed externally, there is currently no API to update the owner.